lib/cc/cli/path_fingerprint.rb

```diff
@@ -0,0 +1,33 @@
+require "digest/md5"
+
+module CC
+  module CLI
+    class PathFingerprint
+      def initialize(output)
+        @output = output
+      end
+
+      def compute
+        md5 = Digest::MD5.new
+        md5 << key
+        md5.hexdigest
+      end
+
+      def key
+        "#{path}|#{check_name}"
```

Comment:
Hmm the more I think about it, I can't really come up with a good example. So, might be fine!